.github/workflows/branch-to-staging.yml

```diff
@@ -12,24 +12,12 @@ jobs:
       build_env: "staging"
       deploy_env: "staging"
       cdk_stack: ${{ vars.WIDGET_STACK_NAME_STAGING }}
-    secrets:
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      AWS_REGION: ${{ secrets.WIDGET_REGION_STAGING }}
-      INFRA_CONFIG: ${{ secrets.INFRA_CONFIG_STAGING }}
-      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+    secrets: inherit
```

Comment:
LGTM - for my own knowledge, is `secrets: inherit` pulling them from the GH Actions Secrets?

Reply:
this means we're passing all secrets available to the current workflow to the downstream one being called here, instead of doing one by one like we were doing before.

since the downstream is our code, I reckoned this is fine and less likely to have a broken CI run